airbyte-webapp/src/packages/cloud/views/layout/SideBar/SideBar.module.scss

```diff
@@ -0,0 +1,43 @@
+@use "../../../../../scss/colors";
```

Comment:
This would be simpler to maintain if we use an [`@forward` rule](https://sass-lang.com/documentation/at-rules/forward) to import the main OSS SideBar styles.  

Then this file can just house the unique classes for the cloud sidebar (I think it's just `.workspaceButton`), but the component can still access the classes from both.

Reply:
Added `@forward ` and disabled rule `css-modules/no-undef-class`